spock-specs/src/test/groovy/org/spockframework/runtime/model/SpecInfoSpec.groovy

```diff
@@ -0,0 +1,39 @@
+package org.spockframework.runtime.model
+
+import org.spockframework.runtime.*
+import spock.lang.*
+import spock.util.EmbeddedSpecCompiler
+
+class SpecInfoSpec extends Specification {
+
+  @Issue("https://github.com/spockframework/spock/issues/1512")
+  def "multiple fixture methods are not supported"() {
+    given:
+    def classes = new EmbeddedSpecCompiler().compile("""import spock.lang.Specification
+class ASpec extends  Specification {
+
+  def $fixtureMethod(String param = 'default') {
+    println param
+  }
+
+  def test() {
+    expect:
+    true
+  }
+}
+""")
```

Comment:
Try this formating style:

```groovy
    def classes = new EmbeddedSpecCompiler().compile("""\
      import spock.lang.Specification
      class ASpec extends Specification {
        def $fixtureMethod(String param = 'default') {
          println param
        }
        def test() {
          expect:
          true
        }
      }
      """.stripIndent())
```

or call `.stripIndent()` always in `.compile()` method, or define new method `.compileIndented()`.

Reply:
White space for the embedded code is irrelevant, but I've changed the indentation